nim_status/lib/account.nim

```diff
@@ -16,3 +17,14 @@ proc createAccount*(rng: ref BrHmacDrbgContext): Account =
     publicKey: $publicKey,
     privateKey: $privateKey
   )
+
+proc derivePubKeyFromPrivateKey*(privateKey: string): string =
+  let privKey = PrivateKey.fromRaw(hexToSeqByte(privateKey)).get()
```

Comment:
fyi, `get` panics / quits the process on failure (if the key is invalid for example)